src/stream/src/common/table/state_table.rs

```diff
@@ -1377,6 +1377,22 @@ where
         Ok(stream.map_ok(|(_, row)| row))
     }
 
+    /// Get the row from a state table with only 1 row.
+    pub async fn get_from_one_row_table(&self) -> StreamExecutorResult<Option<OwnedRow>> {
```

Comment:
I find it a bit ambiguous: having a table with exactly "one row" doesn't necessarily mean the key is empty. If we expect the exactly same behavior as before, a range scan seems still necessary. 🤔

Reply:
I think the case should not exist: if the key is non-empty and has only one row, then it's just a constant. 🤔